l10n_it_sdi_channel/security/security.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+
+    <record model="ir.rule" id="l10n_it_sdi_channel_rule">
+        <field name="name">ES channel multi-company</field>
+        <field name="model_id" ref="model_sdi_channel" />
+        <field name="global" eval="True" />
+        <field
+            name="domain_force"
+        >['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
```

Comment:
Non dovrebbe essere 
['|',('company_id','=',False),('company_id', 'in', company_ids)]
?
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Reply:
Giusto aggiorno